src/benchmarklib/tpcc/procedures/abstract_tpcc_procedure.hpp

```diff
@@ -24,10 +24,10 @@ class AbstractTPCCProcedure {
  protected:
   [[nodiscard]] virtual bool _on_execute() = 0;
 
-  // As random values are generate during creation of the procedure, this is mostly done in a single thread, not in the
+  // As random values are generated during creation of the procedure, this is mostly done in a single thread, not in the
   // database worker's. As such, having a fixed seed for all thread-local random engines should not be an issue.
-  inline static thread_local std::minstd_rand _random_engine{42};
-  inline static thread_local TPCCRandomGenerator _tpcc_random_generator{42};
+  inline static std::minstd_rand _random_engine{17};
+  inline static TPCCRandomGenerator _tpcc_random_generator{17};
```

Comment:
Why the different seed? Though I would not expect that, it could have a tiny influence on the benchmarks.

Reply:
Because the previous seed was obviously wrong.

I'll check numbers later.

Reply:
Sorry, but I do not immediately understand why 42 is obiously wrong, but 17 is not 😅

Reply:
I'll inform Marilena to remove you from anything EPIC-related. Disgraceful!

Reply:
126 would be sth I'd understand